config_parse.go

```diff
@@ -28,6 +30,11 @@ func ReadProxyConfig(path string) (c ProxyConfig, err error) {
 		return
 	}
 
+	err = envconfig.Process("veneur", &c)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
Good Q. In inspecting `Process` the failure modes are one of:
* Require fields are missing (not a problem here)
* The supplied value not being parseable into the field type required

I opted for `Fatal` because if someone is supplying an environment variable with the expectation that it work it seems prudent to bail and let them know it did not work. But this is also a sort of ancillary feature for us so I don't feel strongly. :)